scripts/install_utils.sh

```diff
@@ -124,3 +124,54 @@ install_executorch_libs() {
 
   install_executorch_python_libs $1
 }
+
+clone_torchao() {
```

Comment:
We have a pin for 0.5.0 that you might be able to piggy back off of. Can save you some effort

https://github.com/pytorch/torchchat/pull/1136

Reply:
This is cloning torchao from source, but the pin looks like it is from pip?

Reply:
We can decide on it when your diff lands

Big thing here is that AO doesn't have Mac nightlies so I'm fine with us moving back to direct clones for mac